src/hb-ot-glyf-table.hh

```diff
@@ -81,26 +80,170 @@ struct glyf
     return_trace (true);
   }
 
-  bool subset (hb_subset_plan_t *plan) const
+  template<typename Iterator,
+	   hb_requires (hb_is_source_of (Iterator, unsigned int))>
+  static bool
+  _add_loca_and_head (hb_subset_plan_t * plan, Iterator padded_offsets)
+  {
+    unsigned int max_offset =
+    + padded_offsets
+    | hb_reduce (hb_max, 0);
+    bool use_short_loca = max_offset <= 131070;
```

Comment:
Nitpick: IMHO, hex is more readable/obvious here. I can easily guess what `<= 0x1FFFE` (or even better `< 0x1FFFF`) is about, but not the decimal number used.

Reply:
Ty, I like `< 0x1FFFF`